Auth/Auth.Platform.Android/Authenticator/AndroidAuthenticateOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Android.App;
+using static AeroGear.Mobile.Core.Utils.SanityCheck;
+
+namespace AeroGear.Mobile.Auth.Authenticator
+{
+    public class AndroidAuthenticateOptions: IAuthenticateOptions
+    {
+        /// <summary>
+        /// The activity that the authentication request is triggered from
+        /// </summary>
+        /// <value>an instance of <see cref="Android.App.Activity"></value>
+        public Activity FromActvity { get; private set;  }
```

Comment:
Since property are set only in the constructor and we access them with `this.`, we can remove the private setter